packages/integration-tests/index.js

```diff
@@ -616,14 +763,19 @@ async function rulesList(stackName, bucketName, rulesDirectory) {
  * @returns {Promise.<number>} - Number of rules deleted
  */
 async function deleteRules(stackName, bucketName, rules, postfix) {
+  // setProcessEnvironment is not needed by this function, but other code
```

Comment:
Is this correct? `setProcessEnvironment` is setting the env var for the rules table name, which is needed by this function

Reply:
`RulesTable` is the only env var that is required by this function so, in an ideal world, only that one would be set. Because there is other code (mostly our spec tests) that are implicitly depending on `deleteRules` and the other functions in here to set those other environment variables, `setProcessEnvironment` must be run.

Until it can be refactored, I'd feel better leaving this the way it is.